src/ext/oblivc/obliv_bits.c

```diff
@@ -56,7 +56,6 @@ static struct stdioTransport stdioTransport
 void protocolUseStdio(ProtocolDesc* pd)
   { pd->trans = &stdioTransport.cb; }
 
-//#define PROFILE_NETWORK
```

Comment:
The difference is observable as I can remember. But you can do an experiment on the latest version to make sure.